docs/howto/README.md

```diff
@@ -58,19 +56,13 @@
 ## Helper tools and optimization
 - [How to Leverage Apollo's RTK mode](how_to_leverage_rtk_mode.md)
 - [How to run the Map Data Verification Tool](how_to_run_map_verification_tool.md)
-- [How to run offline perception visualizer](how_to_run_offline_perception_visualizer.md)
-- [How to run offline sequential obstacle perception visualizer](how_to_run_offline_sequential_obstacle_perception_visualizer.md)
 - [How to run perception module on your local computer](how_to_run_perception_module_on_your_local_computer.md)
 - [How to tune control parameters](how_to_tune_control_parameters.md)
 - [How to update vehicle calibration](how_to_update_vehicle_calibration.md)
 - [How to do performance profiling](how_to_do_performance_profiling.md)
-- [How to run Offline Perception Visualizer for Apollo 2.5](how_to_run_apollo_2.5_perception_visualizer.md)
-- [How to use Common Cyber Commands - Basic guide](how_to_use_cyber_commands.md)
```

Comment:
Cyber commands are already provided in docs/cyber. The rather rough introduction here didn't help much.